examples/android/androidAdvanced3DExample/src/Swarm.cpp

```diff
@@ -45,8 +45,8 @@ void Swarm::init(int _nParticles, float positionDispersion, float velocityDisper
 	// SETUP ARRAYS
 	///////////////////////////////////////////
 	//
-	positions = new ofVec3f[nParticles];
```

Comment:
This may be a different project, but we should get rid of heap allocation here and instead use `std::vector<>`